server/eclipse-project/src/test/java/us/freeandfair/corla/util/NaturalOrderComparatorTest.java

```diff
@@ -0,0 +1,22 @@
+package us.freeandfair.corla.util;
+
+import static org.testng.Assert.*;
+import org.testng.annotations.*;
+import java.util.Comparator;
+
+public class NaturalOrderComparatorTest {
+  private Comparator c;
+
+  @BeforeClass()
+  public void NaturalOrderComparatorTest() {
+      this.c = new NaturalOrderComparator();
+  }
+
+  @Test()
+  public void compareTest() {
+      assertEquals(c.compare("Batch 1", "Batch 11"), -1);
+      assertEquals(c.compare("Batch 1", "batch 1"), 0);
+      assertEquals(c.compare("Batch 11", "Batch 2"), 1);
+      assertEquals(c.compare("Batch 123", "Batch 123.a"), -1);
```

Comment:
What happens if you compare "Batch 19" and "Batch 20"?

Reply:
If you compare `"Batch 19"` and `"Batch 20"`, you get `-1`, putting Batch 19 before Batch 20.